test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -35,7 +35,8 @@ gemini_cmd: "gemini -d --duration 2h \
 --max-mutation-retries 5 --max-mutation-retries-backoff 500ms \
 --async-objects-stabilization-attempts 5 --async-objects-stabilization-backoff 500ms \
 --replication-strategy \"{'class': 'NetworkTopologyStrategy', 'replication_factor': '3'}\" \
---table-options \"cdc={'enabled': true}\" --test-username cassandra --test-password cassandra"
+--test-username cassandra --test-password cassandra"
```

Comment:
Is cdc not covered at all? or covered on another upgrade scenario/keyspace?
Do we plan testing both upgrade modes of cdc+vnodes vs. tablets?

Reply:
As i know, now it is impossible to enable cdc on table with tablets due to issue #16317. So i disable it temporary for while it not be fixed

Reply:
I would expect someone from the team handling tablets to have a list of things needed to be tested with tablets disabled

CDC, and rolling upgrades, or a mix of them, probably need to be on this list.

Reply:
@fruch , i think the [following doc](https://docs.google.com/document/d/1WNW0Wo27AK9RoPhU38UHmX9TAzeILuoKv8OX4b7kn4Y/edit?usp=sharing) can help with that.

LWT, CDC, Resharding, and few more.
We can either adjust the current longevities for that or cover it under the new folder of ["tablets_disabled" PR](https://github.com/scylladb/scylla-cluster-tests/pull/7655).

Reply:
@aleksbykov something doesn't add up here, where are you removing cdc from gemini command, and not skipping the gemini command

seems from the code that the gemini command was added for the sake of testing CDC.

now if someone would disable raft and tablets, gemini would still run without cdc.
not sure that the expected outcome.

i.e. one should consider the support matrix of both options, when looking at those changes